codegen/allapigen.py

```diff
@@ -8,7 +8,7 @@
 import settingsgen
 import tuigen
 
-from ansys.fluent.core.launcher.launcher import FluentVersion, get_ansys_version
+from ansys.fluent.core.launcher.launcher_utils import FluentVersion, get_ansys_version
```

Comment:
Please all see PR of @ansaboutin. He has reworked and moved `FluentVersion` into utils.

Reply:
#2262

Reply:
Yes, we will resolve conflicts in @ansaboutin PR after merging this PR. Thank you.